classes/author/Repository.inc.php

```diff
@@ -0,0 +1,26 @@
+<?php
+/**
+ * @file classes/publication/Repository.inc.php
+ *
+ * Copyright (c) 2014-2021 Simon Fraser University
+ * Copyright (c) 2000-2021 John Willinsky
+ * Distributed under the GNU GPL v3. For full terms see the file docs/COPYING.
+ *
+ * @class publication
+ *
+ * @brief Get publications and information about publications
+ */
+
+namespace APP\author;
+
+use APP\core\Application;
+use APP\core\Services;
+use APP\i18n\AppLocale;
+use APP\payment\ojs\OJSPaymentManager;
+use APP\submission\Submission;
+use PKP\core\Core;
+use PKP\db\DAORegistry;
+
+class Repository extends \PKP\author\Repository
+{
+}
```

Comment:
This is not needed. Instead, add `PKP\author\Repository` to the repo facade at `lib/pkp/classes/facades/Repo.inc.php`.

Reply:
Fixed